certified-connectors/LCP - iCordis/apiDefinition.swagger.json

```diff
@@ -0,0 +1,343 @@
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/Export/eloketjson": {
+      "get": {
+        "tags": [
+          "Export"
+        ],
+        "summary": "Get submission(s) for an iCordis e-loket form",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "formid",
+            "description": "id of the form to get the submissions for",
+            "type": "integer",
+            "format": "int32",
+            "required": true,
+            "x-ms-summary": "eloket form id"
+          },
+          {
+            "in": "query",
+            "name": "entryid",
+            "description": "id of the submission to get",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "eloket submission id"
+          },
+          {
+            "in": "query",
+            "name": "lastsynch",
+            "description": "get submissions from this datetime on",
+            "type": "string",
+            "format": "date-time",
+            "x-ms-summary": "date from"
+          },
+          {
+            "in": "query",
+            "name": "includeFiles",
+            "description": "if true, will include the attachments as base64 string",
+            "type": "boolean",
+            "x-ms-summary": "include files"
+          },
+          {
+            "in": "query",
+            "name": "includePDF",
+            "description": "if true, a base64 pdf will be included which contains the form with its values",
+            "type": "boolean",
+            "x-ms-summary": "include PDF of submission"
+          },
+          {
+            "in": "query",
+            "name": "includeHTML",
+            "description": "if true, will contain a base64 html of the form with its values",
+            "type": "boolean",
+            "x-ms-summary": "include HTML of submission"
+          },
+          {
+            "in": "query",
+            "name": "page",
+            "description": "if given, will only return 10 submissions and paging will be enabled. If not given, all submissions will be returned.",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "page"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EloketExport"
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          }
+        },
+        "security": [
+          {
+            "oauth2-auth": [
+              "Export"
+            ]
+          }
+        ],
+        "description": "Get the submissions for a specific form. There is a possibility to pass several parameters to customize the export.",
+        "operationId": "eloket-export"
```

Comment:
@stijn-lcp all operationIds should be in Camel Case. please change to "EloketExport".
Please fix all operationIds in the swagger file.